app/controllers/posts_controller.rb

```diff
@@ -11,4 +11,26 @@ def show
     @user = User.find(@user_id)
     @post = Post.find(@post_id)
   end
+
+  def new; end
+
+  def create
+    # Post.create(author: UsersController.current_user, title: params[:title], text: params[:text])
+
```

Comment:
- [ ] I would recommend removing the commented line to make the codebase cleaner.